### DISCUSSION TITLE
I do not get student benefit yet though I have been approved

### DISCUSSION THREAD

**Discussion by Within-yao** [community/community#148019] [GitHub Education]
### Hi everyone,

I do not get student benefit yet though I have been approved.How long will it take for me to receive student benefits?

**Comment by CyberSphinxxx** [comment]
Hi @Within-yao,

Once your application is approved, it can take a few days for the student benefits to activate. If you've been waiting longer, try logging out and back into your account to refresh the status. If the issue persists, consider reaching out to GitHub Education Support [here](https://support.github.com/contact/education) for help.

**Reply by CyberSphinxxx** [reply]
I checked your GitHub account and can see that there is a Pro badge on your profile, which means you already have access to the GitHub Student Developer Pack benefits.

**Reply by Within-yao** [reply]
Thank you for your replying！

**Reply by Binara-99** [reply]
I have this issue too. how to fix that help me 
![Screenshot 2024-12-30 080351](https://github.com/user-attachments/assets/6ea4392b-9593-4112-8fe2-8a1acf5d4a15)

**Reply by CyberSphinxxx** [reply]
@Binara-99, You may have to wait for it. You will have it soon as it was accepted recently.
